### PR TITLE
Add QE POC nightly testing trigger definition.

### PR DESCRIPTION
**Pull request by tremes**
@trepel I didn't create an issue but rather a PR

### CODE REVIEW

File: jobs/integr8ly/poc-testing-nightly-trigger.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            GH_CLIENT_ID=yourclientid
+            GH_CLIENT_SECRET=yourclientsecret
+            SELF_SIGNED_CERTS=false
+            RECIPIENTS=integreatly-qe@redhat.com
+            INSTALLATION_REPOSITORY=https://github.com/integr8ly/installation.git
+            INSTALLATION_BRANCH=master
+            TEST_SUITES_REPOSITORY=https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
+            TEST_SUITES_BRANCH=master
+            CLUSTER_URL=https://integreatly-qe.opentry.me
+            MASTER_URLS=ec2-54-154-4-117.eu-west-1.compute.amazonaws.com,ec2-34-244-30-214.eu-west-1.compute.amazonaws.com,ec2-34-241-180-41.eu-west-1.compute.amazonaws.com
```
Comment (pawelpaszki) [inline jobs/integr8ly/poc-testing-nightly-trigger.yaml]: I think master URLs might be outdated after recent changes. Also (if my PR is merged) there will be one additional parameter required: `TO_DO=full`.

File: jobs/integr8ly/poc-testing-nightly-trigger.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    triggers:
+      - timed: "H H(0-3) * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+      - inject:
+          properties-content: |
+            GH_CLIENT_ID=yourclientid
+            GH_CLIENT_SECRET=yourclientsecret
+            SELF_SIGNED_CERTS=false
```
Comment (pawelpaszki) [inline jobs/integr8ly/poc-testing-nightly-trigger.yaml]: Is the self signed certs suppose to be false (I don't know, hence the question)?

**Comment by pawelpaszki** [comment]
apart from the comments looks good

File: jobs/integr8ly/poc-testing-nightly-trigger.yaml (hunk 1 above)
Comment (tremes) [reply jobs/integr8ly/poc-testing-nightly-trigger.yaml]: yeah sure....

File: jobs/integr8ly/poc-testing-nightly-trigger.yaml (hunk 2 above)
Comment (tremes) [reply jobs/integr8ly/poc-testing-nightly-trigger.yaml]: yes it is. QE POC cluster does have a valid cert.

File: jobs/integr8ly/poc-testing-nightly-trigger.yaml (hunk 1 above)
Comment (tremes) [reply jobs/integr8ly/poc-testing-nightly-trigger.yaml]: Updated to the new cluster

**Pull request merged**